specification/ingest/_types/Processors.ts

```diff
@@ -284,59 +840,185 @@ export enum JsonProcessorConflictStrategy {
+   */
   field: Field
+  /**
+   * Regex pattern to use for splitting key-value pairs.
+   */
   field_split: string
+  /**
+   * If `true` and `field` does not exist or is `null`, the processor quietly exits without modifying the document.
+   * @server_default false
+   */
   ignore_missing?: boolean
+  /**
+   * List of keys to filter and insert into document.
+   * Defaults to including all keys.
+   */
   include_keys?: string[]
+  /**
+   * Prefix to be added to extracted keys.
+   * @server_default null
+   */
   prefix?: string
+  /**
+   * If `true`. strip brackets `()`, `<>`, `[]` as well as quotes `'` and `"` from extracted values.
+   * @server_default false
+   */
   strip_brackets?: boolean
+  /**
+   * The field to insert the extracted keys into.
+   * Defaults to the root of the document.
+   * Supports template snippets.
+   */
   target_field?: Field
+  /**
+   * String of characters to trim from extracted keys.
+   */
   trim_key?: string
+  /**
+   * String of characters to trim from extracted values.
+   */
   trim_value?: string
+  /**
+   * Regex pattern to use for splitting the key from the value within a key-value pair.
+   */
   value_split: string
 }
 
 export class LowercaseProcessor extends ProcessorBase {
+  /**
+   * The field to make lowercase.
+   */
   field: Field
+  /**
+   * If `true` and `field` does not exist or is `null`, the processor quietly exits without modifying the document.
+   * @server_default false
+   */
   ignore_missing?: boolean
+  /**
+   * The field to assign the converted value to, by default field is updated in-place.
+   * @server_default field
+   */
   target_field?: Field
 }
 
 export class PipelineProcessor extends ProcessorBase {
+  /**
+   * The name of the pipeline to execute.
+   * Supports template snippets.
+   */
   name: Name
+  /**
+   * Whether to ignore missing pipelines instead of failing.
+   * @server_default false
+   */
   ignore_missing_pipeline?: boolean
 }
 
 export class RemoveProcessor extends ProcessorBase {
+  /**
+   * Fields to be removed. Supports template snippets.
+   */
   field: Fields
+  /**
+   * If `true` and `field` does not exist or is `null`, the processor quietly exits without modifying the document.
+   * @server_default false
+   */
   ignore_missing?: boolean
 }
 
 export class RenameProcessor extends ProcessorBase {
+  /**
+   * The field to be renamed.
+   * Supports template snippets.
+   */
   field: Field
+  /**
+   * If `true` and `field` does not exist, the processor quietly exits without modifying the document.
+   * @server_default false
+   */
   ignore_missing?: boolean
+  /**
+   * The new name of the field.
+   * Supports template snippets.
+   */
   target_field: Field
 }
 
 export class ScriptProcessor extends ProcessorBase {
+  /**
+   * ID of a stored script.
+   * If no `source` is specified, this parameter is required.
+   */
   id?: Id
+  /**
+   * Script language.
+   * @server_default "painless"
```

Comment:
```suggestion
   * @server_default painless
```

Reply:
I *think* we should not use quotes for string `@server_default`s, based on other examples I've seen.